### PR TITLE
feature(bank-sdk): Transaction List. Transaction Summary.

### PR DESCRIPTION
**Pull request by ndubkov-distcotech**

### CODE REVIEW

File: capture-sdk/sdk/src/main/res/layout/gc_fragment_analysis.xml [hunk 1]
```diff
@@ -131,4 +132,13 @@
         </LinearLayout>
 
     </LinearLayout>
+
```
Comment (abolfazlimahdi) [inline capture-sdk/sdk/src/main/res/layout/gc_fragment_analysis.xml:135]: [Request] Please add gc at the beginning of ids in capture-sdk

File: capture-sdk/sdk/src/main/res/layout/gc_fragment_analysis.xml [hunk 2]
```diff
@@ -131,4 +132,13 @@
...
+    <androidx.compose.ui.platform.ComposeView
+        android:id="@+id/compose_view"
```
Comment (abolfazlimahdi) [inline capture-sdk/sdk/src/main/res/layout/gc_fragment_analysis.xml:137]: [Request] Please apply changes to tablet layouts too.

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/network/AnalysisResult.java [hunk 3]
```diff
@@ -31,9 +31,11 @@ public class AnalysisResult extends Result {
      * @param giniApiDocumentId the id of a document in the Gini API
      * @param extractions       the extractions from the Gini API
      */
-    public AnalysisResult(@NonNull final String giniApiDocumentId,
+    public AnalysisResult(
```
Comment (abolfazlimahdi) [inline capture-sdk/sdk/src/main/java/net/gini/android/capture/network/AnalysisResult.java:34]: [Suggestion] Are these breaking changes? As I see, these are public classes which might be used by clients. Please let me know what you think.

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/network/Result.java [hunk 4]
```diff
@@ -12,14 +12,16 @@
 public class Result {
 
     private final String giniApiDocumentId;
+    private final String giniApiDocumentFilename;
```
Comment (abolfazlimahdi) [inline capture-sdk/sdk/src/main/java/net/gini/android/capture/network/Result.java:16]: [Suggestion] Are these breaking changes? As I see, these are public classes which might be used by clients. Please let me know what you think.

**Review by abolfazlimahdi — CHANGES_REQUESTED** [review]
Thank you. In general, PR looks good. I added some comments to the PR. Also there are some unused imports in the files. Please remove them as well.

File: capture-sdk/sdk/src/main/res/layout/gc_fragment_analysis.xml (hunk 1 above)
Comment (ndubkov-distcotech) [reply capture-sdk/sdk/src/main/res/layout/gc_fragment_analysis.xml]: This component was deleted

File: capture-sdk/sdk/src/main/res/layout/gc_fragment_analysis.xml (hunk 2 above)
Comment (ndubkov-distcotech) [reply capture-sdk/sdk/src/main/res/layout/gc_fragment_analysis.xml]: This component was deleted

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/network/AnalysisResult.java (hunk 3 above)
Comment (ndubkov-distcotech) [reply capture-sdk/sdk/src/main/java/net/gini/android/capture/network/AnalysisResult.java:34]: I'm not sure. We can create extra constructor with some empty text by default. 
Customers which will not use transaction docs will not be affected then

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/network/Result.java (hunk 4 above)
Comment (ndubkov-distcotech) [reply capture-sdk/sdk/src/main/java/net/gini/android/capture/network/Result.java:16]: I'm not sure. We can create extra constructor with some empty text by default. 
Customers which will not use transaction docs will not be affected then

File: bank-sdk/example-app/build.gradle.kts [hunk 5]
```diff
@@ -189,6 +189,9 @@ dependencies {
     implementation(libs.hilt.library)
     implementation(libs.navigation.fragment.ktx)
     implementation(libs.navigation.ui.ktx)
+    implementation(libs.material)
```
[the remark this answers is not in the document]
Comment (abolfazlimahdi) [reply bank-sdk/example-app/build.gradle.kts]: [Suggestion] There are two other libraries added below this. Please double check if we need them in the example app?

File: bank-sdk/sdk/src/main/res/layout-sw600dp-land/gbs_fragment_digital_invoice.xml [hunk 6]
```diff
@@ -172,4 +172,13 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent" />
 
+    <androidx.compose.ui.platform.ComposeView
+        android:id="@+id/compose_view"
```
Comment (abolfazlimahdi) [inline bank-sdk/sdk/src/main/res/layout-sw600dp-land/gbs_fragment_digital_invoice.xml:176]: [Request] Please add gbs prefix to these ids (there are 4 xml files with this id).

**Review by abolfazlimahdi — APPROVED** [review]
Everything looks great. I added one more comment. Thank you!

File: bank-sdk/sdk/src/main/res/layout-sw600dp-land/gbs_fragment_digital_invoice.xml (hunk 6 above)
Comment (ndubkov-distcotech) [reply bank-sdk/sdk/src/main/res/layout-sw600dp-land/gbs_fragment_digital_invoice.xml]: Done

**Pull request merged**